ext/zend_test/test.c

```diff
@@ -364,6 +365,67 @@ static ZEND_FUNCTION(zend_test_crash)
 	php_printf("%s", invalid);
 }
 
+zend_mm_heap* zend_test_heap;
```

Comment:
These probably have to be module globals and properly reset when the request ends.

Reply:
You are right, moved them to the `zend_test` globals.